docs/installation.rst

```diff
@@ -4,7 +4,10 @@ Installation
 Requirements
 ------------
 
-Python 3.7 or higher.
+Python 3.8 or 3.9 and ``asyncio``.
+
+Tested on Linux and macOS. Core code will work anywhere where ``asyncio`` works, however functional tests require a POSIX system.
```

Comment:
why can't tests run on windows?

Reply:
Because `go run foo.go` is weird and I have to start/watch/kill it in a peculiar way.